File: 06 - Java/09-02-00-SubString-ULP.java
public class ProgramaPrincipal {

    public static void main(String[] args){
        var cadena = "Hola Mundo";
        // subcadena hola y mundo
        //substring(indice_inicio, indice_fin - 1)
        var subcadena1 = cadena.substring(0,4);
        System.out.println("Subcadena1 = " + subcadena1);
        var subcadena2 = cadena.substring(5,10);
        System.out.println("Subcadena2 = " + subcadena2);
    }
}

File: 03 - C++/09-02-00-SubString-ULP.cpp
#include <iostream>
using namespace std;

int main() {
	string cadena = "Hola Mundo";
	// subcadena hola y mundo
	//string substr (size_t pos, size_t len);
	// inicio, cantidad de caracteres a copiar
	string subcadena1 = cadena.substr(0,4);
	cout << "Subcadena1 = " << subcadena1 << endl;
	string subcadena2 = cadena.substr(5,5);
	cout << "Subcadena2 = " << subcadena2 << endl;
	
	
	return 0;
}

